egs/wsj/s5/steps/nnet3/make_bottleneck_features.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+#           2016  Pegah Ghahremani
+# Apache 2.0
+# This script dumps bottleneck feature for model trained using nnet3.
+
+# Begin configuration section.
+stage=1
+nj=4
+cmd=queue.pl
+use_gpu=false
+bnf_name=tdnn_bn.renorm # the component-node name in nnet3 model used for
```

Comment:
this is a strange default name to have in a script that's supposed to be generic.
i think it would be better to make it a mandatory positional argument to the script.